AEPNotificationContent/Sources/AEPNotificationViewController.swift

```diff
@@ -39,6 +39,8 @@ open class AEPNotificationViewController: UIViewController, UNNotificationConten
             controller = BasicTemplateController(withPayload: basicPayload, delegate: self)
         } else if let carouselPayload = payload as? CarouselPayload {
             controller = CarouselTemplateController(withPayload: carouselPayload, delegate: self)
+        } else if let timerPayload = payload as? TimerPayload {
```

Comment:
any reason we aren't doing a switch on the templateType here instead of this if-else-if?

Reply:
No reason in particular. 
Reading more we can use pattern matching with switch statements and make these conditions more readable. 
Will change it